tests/test_pkey.py

```diff
@@ -436,3 +445,33 @@ def test_stringification(self):
         key = RSAKey.from_private_key_file(test_path('test_rsa.key'))
         comparable = TEST_KEY_BYTESTR_2 if PY2 else TEST_KEY_BYTESTR_3
         self.assertEqual(str(key), comparable)
+
+    def test_BestAvailableEncryption(self):
+        # the ok case:
+        self.assertTrue(BestAvailableEncryption(b"password"))
+
+        # Doesnt work on 2.6, 2.7 & pypy
+        # bad: not a string
+        # with self.assertRaises(ValueError):
+        #     BestAvailableEncryption("not bytes")
```

Comment:
This commented-out test is no longer applicable.

Also, I don't think we need to unit-test a function that isn't in paramiko.